conf/sql/database-create-postgres.sql

```diff
@@ -2269,3 +2269,12 @@ INSERT INTO pgagent.pga_schedule(
     ARRAY[false, false, false, false, false, false, false, false, false, false, false, false]::boolean[]
 ) RETURNING jscid INTO scid;
 END;
+
+--63.sql
+--- !Downs
+
+ALTER TABLE "public"."subscription" DROP COLUMN default_identity;
```

Comment:
The !Downs are used by play evolution to undo what the ups in the evolution script propose, so in this case, this should be on ups even if it drops a column.